Dockerfile

```diff
@@ -111,12 +113,30 @@ DEBIAN_FRONTEND=noninteractive apt-get purge -y --allow-unauthenticated \
     composablekernel-dev \
     miopen-hip
 
+# TODO: it should be able to automatically get commit hash from requirements.txt
+ARG CK_COMMIT=467b4e502d1c2ee2c5fe85ff9fd637b04a5b7ba7
+RUN wget -O ck.tar.gz https://www.github.com/ROCm/composable_kernel/archive/${CK_COMMIT}.tar.gz && \
+    tar zxvf ck.tar.gz &&\
+    cd composable_kernel-${CK_COMMIT} && \
+    mkdir build && cd build && \
+    CXX=/opt/rocm/bin/amdclang++ cmake \
+    -D CMAKE_PREFIX_PATH=/opt/rocm \
+    -D CMAKE_CXX_COMPILER_LAUNCHER="${COMPILER_LAUNCHER}" \
+    -D CMAKE_BUILD_TYPE=Release \
+    -D GPU_ARCHS="gfx908;gfx90a;gfx942;gfx1100" \
```

Comment:
We have to hard-fuse it instead of passing the variable. Because dockerfile processed variable strings differently, and we will meet errors like:

>Performing Test COMPILER_HAS_TARGET_ID__gfx908 - Failed

hence the extra "_" before "gfx908"